### PR TITLE
feat: added auth page

### PR DESCRIPTION
**Pull request by tanishq20**
Added auth

- Signup page
- Login page
screencapture-localhost-3000-login-2022-04-06-22_58_44

### CODE REVIEW

File: src/components/auth/LoginForm.js [hunk 1]
```diff
@@ -0,0 +1,195 @@
+import { Link } from 'react-router-dom'
+import { useState, useEffect } from 'react'
+import style from './Auth.module.css'
+import { useNavigate } from 'react-router-dom'
```
Comment (rahulcs754) [inline src/components/auth/LoginForm.js]: you can set useNavigate into first line. no need this line

File: src/components/auth/SignupForm.js [hunk 2]
```diff
@@ -0,0 +1,322 @@
+import { useNavigate } from 'react-router-dom'
+import { Link } from 'react-router-dom'
```
Comment (rahulcs754) [inline src/components/auth/SignupForm.js]: same above

**Review by rahulcs754 — APPROVED** [review]
The code looks good and clean 
just small suggestion

**Review by coldpigli — APPROVED** [review]
Looks good and clean. Liked the naming conventions of CSS

File: src/components/auth/LoginForm.js (hunk 1 above)
Comment (tanishq20) [reply src/components/auth/LoginForm.js]: Thanks for pointing it out.

File: src/components/auth/SignupForm.js (hunk 2 above)
Comment (tanishq20) [reply src/components/auth/SignupForm.js]: Thanks for pointing it out.

**Pull request merged**